docs/adr/0005-keep-document-history-in-sync-with-rabbit-mq.md

```diff
@@ -0,0 +1,64 @@
+# 5. Keep Document history in sync with RabbitMQ
```

Comment:
I wonder if it's worth grouping or labelling this and our last ADR somehow, to show that these are changes relating specifically to the content block manager? Or referencing them in our engine? Just aware that if the engine was moved out of Whitehall they might lose their context

Reply:
I think in this case, it's more related to actual Whitehall code. I do think anything related directly to Content Block Manager should go elsewhere though